/*
const promise = new Promise((resolve, reject) => {
    setTimeout(()=>{
   //     resolve('this is my data');
        reject('Errors')
    }, 5000);
});
promise
    .then((data) => {
        console.log(data);
    }).catch((data) => {
        console.log(data)
    });*/
